web/modules/custom/dpl_loans/dpl_loans.module

```diff
@@ -19,3 +21,57 @@ function dpl_loans_dpl_react_apps_data(array &$data): void {
     ),
   ];
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * @param array<mixed> $form
+ *   An associative array containing the structure of the form.
+ * @param \Drupal\Core\Form\FormStateInterface $form_state
+ *   The current state of the form.
+ */
+function dpl_loans_form_dpl_library_agency_list_size_settings_form_alter(array &$form, FormStateInterface $form_state): void {
+  $config = \Drupal::service('config.factory')
+    ->getEditable('dpl_library_agency.list_size_settings');
+
+  $form['loan_list_size_settings'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Loan list size settings', [], ['context' => 'List size (settings)']),
+    '#description' => t('The number of items to display in the loan list.', [], ['context' => 'List size (settings)']),
+    '#tree' => FALSE,
+  ];
+
+  $form['loan_list_size_settings']['loan_list_size_desktop'] = [
+    '#type' => 'number',
+    '#title' => t('Loan list size on desktop', [], ['context' => 'List size (settings)']),
+    '#default_value' => $config->get('loan_list_size_desktop') ?? DplLoansSettings::LOAN_LIST_SIZE_DESKTOP,
+    '#min' => 1,
+    '#step' => 1,
+  ];
+
+  $form['loan_list_size_settings']['loan_list_size_mobile'] = [
+    '#type' => 'number',
+    '#title' => t('Loan list size on mobile', [], ['context' => 'List size (settings)']),
+    '#default_value' => $config->get('loan_list_size_mobile') ?? DplLoansSettings::LOAN_LIST_SIZE_MOBILE,
+    '#min' => 1,
+    '#step' => 1,
+  ];
+
+  $form['#submit'][] = 'dpl_loans_handler_submit';
+}
+
+/**
+ * Custom submit callback.
+ *
+ * @param array<mixed> $form
+ *   An associative array containing the structure of the form.
+ * @param \Drupal\Core\Form\FormStateInterface $form_state
+ *   The current state of the form.
+ */
+function dpl_loans_handler_submit(array &$form, FormStateInterface $form_state): void {
+  \Drupal::service('config.factory')
```

Comment:
You should use config service here instead.

Reply:
I tried implementing the use of config service here, but only with limited success.
Config service will not let me use `->set()` function, as the config is immutable.

Do you have another suggestion?